cunumeric/module.py

```diff
@@ -3506,6 +3506,58 @@ def put(
     a.put(indices=indices, values=values, mode=mode)
 
 
+def _can_broadcast_to_shape(self: ndarray, shape: NdShape) -> bool:
```

Comment:
can't we just use `np.broadcast_shapes` instead of this custom code?

Reply:
sure, will use it instead with `try\ except"